src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditorViewModel.java

```diff
@@ -196,6 +196,7 @@ public void bindToEntry(BibEntry entry) {
                         preferences.getFilePreferences(),
                         externalFileTypes);
                 newLinkedFile.markAsAutomaticallyFound();
```

Comment:
Move `newLinkedFile.setOpacityProperty(0.3);` to `markAsAutomaticallyFound` so that it is consistent, even if some other code marks a file as automatically found.